crypto/benchmarking/bench.go

```diff
@@ -0,0 +1,88 @@
+package benchmarking
```

Comment:
`crypto/` already has some folders (hd, keyring, ledger). Would it make more sense to put all these files into `crypto/pubkey/`?

Reply:
`crypto/(key, keys)` because it houses private keys as well

Reply:
I didn't have a preference here, either way seems fine to me. Will move to `crypto/keys` if that seems better

Reply:
I also prefer `keys` than `key`.